libs/api/domains/license-service/src/lib/mappers/firearmLicenseMapper.ts

```diff
@@ -21,16 +21,27 @@ import { FormatMessage, IntlService } from '@island.is/cms-translations'
 import { m } from '../messages'
 import { expiryTag, formatDate } from '../utils'
 import { GenericLicenseDataField } from '../dto/GenericLicenseDataField.dto'
+import { UserAgent } from '@island.is/nest/core'
+import { enableAppCompatibilityMode } from '../utils/appCompatibilityMode'
+
+const APP_VERSION_CUTOFF = '1.4.7'
```

Comment:
I haven't tried it yet - waiting for feature deploy to work, but was the idea to release this in the next release (26th of November) or hotfix it sooner? If it will not go out until 26th then I would suggesting changing this to 1.4.8 since I will be releasing 1.4.7 next week 😊

Reply:
I think having it in an official release would be good, i'll change the version